modules/understand-admission-controller-enforcement.adoc

```diff
@@ -24,4 +24,5 @@ Also, consider API timeouts in any custom automation you might be using.
 To resolve this issue, delete the `ValidatingWebhookConfiguration` object as described in the disabling admission controller enforcement section.
 //link to Disable admission controller enforcement
 * If you have deploy-time enforcement enabled for a policy and you enable the admission controller, {product-title-short} attempts to block deployments that violate the policy.
++ [NOTE] ==== `Reasses all` will not scale down existing deployments. ====
```

Comment:
will not apply enforcement, if enabled on policies, to existing deployments. (trying to keep it generic to enforcement like above sentence)

Reply:
Agree and done